remedy.lic

```diff
@@ -67,7 +67,11 @@ class Remedy
 
     get_item(@container)
     if ['red flower', 'blue flower'].include?(@herb1)
-      @herb1 = 'dried flower'
+      /You tap (.*) inside your|I could not find|You lightly tap/ =~ bput("tap #{@herb1} in my #{@bag}", 'You tap (.*) inside your', 'I could not find', 'You lightly tap')
```

Comment:
Check out the `exists?` method in `common-items`. It taps something and returns true if it exists. So `exists?('red flower')` would return true if it was tapped or false if it saw some other message.